prettier.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  arrowParens: 'always',
+  bracketSpacing: false,
+  semi: true,
+  singleQuote: true,
+  trailingComma: 'all',
```

Comment:
I'm not sure, if it is good to have two different stylers applied sequentially, because one will override the previous and might lead to failing eslint checks on PRs. If we try to apply such a mechanism, the prettier configuration should exactly mirror the eslint configuration:
```
 indent: [ error, 2, {
    MemberExpression: off
  } ]
  linebreak-style: [ error, unix ]
  semi: [ error, always ]
  quotes: [ error, single, { avoidEscape: true } ]
  no-mixed-spaces-and-tabs: error
  space-before-blocks: error
  arrow-spacing: error
  key-spacing: [ error, { afterColon: true, mode: minimum } ]
  brace-style: [ error, '1tbs' ]
  comma-spacing: [ error, { before: false, after: true } ]
  comma-style: [ error, last, { exceptions: { VariableDeclaration: true } } ]
  array-bracket-spacing: [ error, always, { singleValue: false } ]
  computed-property-spacing: [ error, never ]
  object-curly-spacing: [ error, always ]
  prefer-const: error
  no-var: error
  promise/no-nesting: off
  import/first: error
  import/newline-after-import: error
  import/no-named-as-default: off
  import/no-extraneous-dependencies: [ error, { devDependencies: true } ]
  lodash/import-scope: off
  lodash/preferred-alias: off
  lodash/prop-shorthand: off
  lodash/prefer-lodash-method: [ error, { ignoreObjects: [ BbPromise, path ] } ]
```
Currently it does not. Eslint applies much more and different restrictions even for different imported modules.
Maybe it would be enough to just run `eslint --fix` but not prettier?

Reply:
Prettier doesn't come with a ton of options unfortunately (https://prettier.io/docs/en/options.html). That said, I wouldn't worry about ESLint/Prettier conflicting, since I've also added [`eslint-config-prettier`](https://github.com/prettier/eslint-config-prettier), which exists solely for this purpose.

Reply:
We use something very similar to this and it works great for us where I work. We don't run into issues with the two conflicting because of the packages mentioned above.

Reply:
Ok, you convinced me 😄 .